app/Http/Controllers/RoomController.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\Room;
+use Illuminate\Http\Request;
+
+class RoomController extends Controller
+{
+    public function validatedData()
+    {
+        return request()->validate([
```

Comment:
Please add more validations to the fields(if required by out data types and etc.)
Example: if there's a email field, the validation should something like this `email` => 'required|email'
Reference: https://laravel.com/docs/9.x/validation#introduction